app/components/modal_component.html.erb

```diff
@@ -1,14 +1,11 @@
-<%= content_tag(:'lg-modal', **tag_options, class: css_class, hidden: true) do %>
-  <%= tag.div(
-        role: 'dialog',
-        class: 'usa-modal-overlay',
+<%= content_tag(:'lg-modal', **tag_options) do %>
+  <%= tag.dialog(
```

Comment:
idk if it matters, but seems goofy to me to use the two different tag builder syntaxes, could we do `content_tag` for both? I think then we could even pass in `content` directly instead if needing to put it in a block

```erb
<%= content_tag(:'lg-modal', **tag_options) do %>
  <%= content_tag(
        :dialog,
        content,
        aria: { ... },
      ) %>
<% end %>
```

```

Reply:
Yeah agree it's weird. Generally I'd prefer `tag.` syntax as it's a little more succinct (readable?), though dunno that it works with the custom element tag 🤔 Either way, I'll align.

Reply:
Updated in 11dfb66e7.